### PR TITLE
Feature/metrics

### PR DESCRIPTION
**Pull request by cympfh**

### CODE REVIEW

File: sources/metrics.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```python
+from keras import metrics
+
+model.compile(loss='mean_squared_error',
+              optimizer='sgd',
+              metrics=[metrics.mae, metrics.categorical_accuracy])
+```
+
+評価関数は [目的関数](/objectives) とよく似ていますが，
```
Comment (nzw0301) [inline sources/metrics.md]: v2からは`objectives` ではなく`losses`になっていますので変更お願いします．

File: sources/metrics.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+渡す `metrics` パラメータには既存の評価関数の名前を引数に与えるか，
+自分で作った評価関数を渡す事ができます ([カスタマイズ](#_3) を参照して下さい)．
+
+## 利用可能な評価関数
+
+- __binary_accuracy__
+- __categorical_accuracy__
+- __sparse_categorical_accuracy__
+- __top_k_categorical_accuracy__
```
Comment (nzw0301) [inline sources/metrics.md]: v2のdocsから評価関数は引数付きで出されているので対応お願いします． 例: https://keras.io/metrics/#binary_accuracy

File: sources/metrics.md (hunk 1 above)
Comment (cympfh) [reply sources/metrics.md]: 古いものを参照していました :bow:
修正しました

**Comment by nzw0301** [comment]
ありがとうございます！

**Review by nivekpraht — APPROVED** [review]
